docs/documentation.md

```diff
@@ -0,0 +1,283 @@
+# qtop.py guide
+
+  * [Introduction](#introduction)
+  * [Quickstart](#quickstart)
+  * [Output walkthrough](#output-walkthrough)
+  * [Watchmode](#watch-mode)
+  * [Customisation](#customisation)
+  * [Command-line arguments]() (TODO)
+  * [Usage tips]() (TODO)
+
+## Introduction
+
+### Background
+
+qtop.py is based on the same-titled bash script (well, minus the .py!) written by @fgeorgatos long ago. The main reason for rewriting it from scratch in python was the drive to add some much-needed functionality, which was getting harder and harder to do in bash. The new version of qtop.py is much more extendable, so as to be able to cooperate with more Local Resource Management Systems (LRMSs), such as Oracle/Sun Grid Engine (SGE/GE), LSF, LoadLeveler, OAR, SLURM etc. It's also less buggy, with most of the bugs of the old version eradicated.
+
+### Goals
+
+qtop.py is a tool written to summarize, textually and visually, the state of an LRMS, along with some relevant information around clusters, Grid-specific or independent. It gathers information from the LRMS and organises them in a compact yet illustrative format, for easy access. 
```

Comment:
's/LRMS/Resource Management System/g'

ie. we can hint on the ability to visualise more generic resource types...